berkdb/btree/bt_search.c

```diff
@@ -227,7 +227,7 @@ genid_hash_init(DB_ENV *dbenv, int szkb)
 		}
 	}
 
-	pthread_mutex_init(&(h->mutex), NULL);
+	 pthread_mutex_init(&(h->mutex), NULL);
```

Comment:
?

Reply:
? ?

Reply:
Was to test the warning above. It works.

Reply:
IC

Reply:
THX